kolibri/core/assets/src/core-app/index.js

```diff
@@ -1,58 +1,100 @@
-// Shim Array includes for ES7 spec compliance
-require('array-includes').shim();
+/**
+ * Provides the public API for the Kolibri FrontEnd core app.
+ * @module Facade
+ */
 // polyfill for older browsers
 // TODO: rtibbles whittle down these polyfills to only what is needed for the application
-require('core-js');
-require('url-polyfill');
-
+import 'core-js';
+// Required to setup Keen UI, should be imported only once in your project
+import 'keen-ui/src/bootstrap';
+import '../keen-config/font-stack.scss';
+import '../styles/main.scss';
+import urls from 'kolibri.urls';
+import * as theme from 'kolibri-components/src/styles/theme';
+import generateGlobalStyles from 'kolibri-components/src/styles/generateGlobalStyles';
+import trackInputModality from 'kolibri-components/src/styles/trackInputModality';
+import keenUiConfig from 'keen-ui/src/config';
+import branding from 'kolibri.utils.branding';
+import logging from 'kolibri.lib.logging';
+import Vue from 'vue';
+import VueMeta from 'vue-meta';
+import VueRouter from 'vue-router';
+import Vuex from 'vuex';
+import KThemePlugin from 'kolibri-components/src/KThemePlugin';
+import heartbeat from 'kolibri.heartbeat';
+import KContentPlugin from 'kolibri-components/src/content/KContentPlugin';
+import keenOptions from '../keen-config/options.json';
+import { i18nSetup, languageDirection } from '../utils/i18n';
+import ContentRendererLoadingComponent from '../views/ContentRenderer/ContentRendererLoading';
+import ContentRendererErrorComponent from '../views/ContentRenderer/ContentRendererError';
+import apiSpec from './apiSpec';
+import monitorPageVisibility from './monitorPageVisibility';
+import getPluginData from 'kolibri.utils.getPluginData';
```

Comment:
Generally I use API spec style imports for anything exposed on the core API.

`getPluginData` is a little weird, because I set it up to be referenced using the core API path, but it is actually imported at build time (because it needs build time information to properly function).